test/issue_97_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule Issue97Test do
+  use ExUnit.Case
+
+  test "stream_tags doesn't hang when used with stream take on empty xml" do
```

Comment:
```suggestion
defmodule SweetXmlTest do
  use ExUnit.Case

  describe "stream_tags/3" do
    # c.f. https://github.com/kbrw/sweet_xml/issues/97
    test "should not hang when used with Stream.take/2 on an empty xml" do
```

:nail_care: 
While I do like the fact that this helps link these tests to the issue raised to gain an understanding of the reason it was added, i think having a meaningful test module name might be better for overall comprehension and reading of test files.
Especially considering there are several ways of achieving the same results through other means; for instance, the commit itself already have that information. We could even put a link to the Github issue in the commit description, and / or even in a `@doc` or `@docmodule` attribute here.